zerver/models.py

```diff
@@ -2226,6 +2226,12 @@ class PreregistrationUser(models.Model):
     )
     invited_as: int = models.PositiveSmallIntegerField(default=INVITE_AS["MEMBER"])
 
+    # The UserProfile created upon completion of the registration
+    # for this PregistrationUser
+    created_user_profile: Optional[UserProfile] = models.ForeignKey(
```

Comment:
Maybe we should call this `created_user`? I think we are planning to eventually remove the `_profile` from the name in database tables.